hypothesis-python/tests/conjecture/test_data_tree.py

```diff
@@ -39,14 +39,14 @@
     run_to_nodes,
 )
 
-TEST_SETTINGS = settings(
-    max_examples=5000, database=None, suppress_health_check=list(HealthCheck)
+runner_settings = settings(
+    max_examples=100, database=None, suppress_health_check=list(HealthCheck)
```

Comment:
I'm skeptical of the blanket health check suppression here, but I'm leaving that for another day